wiki/Game_Modifiers/en.md

```diff
@@ -646,13 +646,13 @@ It does not affect the notes' timeline and is not shuffled every time Random mod
 
 > \*\* UNRANKED \*\*
 
-[Introduced in 22 February 2017 (2017-02-22) build \[b20170222.3 (Stable)\]](https://osu.ppy.sh/p/changelog?v=b20170222.3 "Release Notes for b20170222.3 (Stable)"), ScoreV2 mod enables the prototype ScoreV2 system over the current scoring system used by the mode which would be used in _osu!_ in the future to replace the current individual scoring system by modes. ScoreV2 system attempt to scale _all_ hit object scoring values to a hard limit of 1,000,000 (1 million) score limit at 1.00 score modifier excluding bonus spinner value of spinners in osu!standard mode, dendens in osu!taiko mode, and collected bananas (spinner equivalent) in osu!catch mode.
+[Introduced in 22 February 2017 (2017-02-22) build \[b20170222.3 (Stable)\]](https://osu.ppy.sh/p/changelog?v=b20170222.3 "Release Notes for b20170222.3 (Stable)"), ScoreV2 mod uses the prototype ScoreV2 system instead of the current scoring system used by the mode which would be used in _osu!_ in the future to replace the current individual scoring system by modes. ScoreV2 system attempts to scale _all_ hit object scoring values to a hard limit of 1,000,000 (1 million) score limit at 1.00 score modifier excluding bonus spinner value of spinners in osu!standard mode, dendens in osu!taiko mode, and collected bananas (spinner equivalent) in osu!catch mode.
```

Comment:
* \+ 'the' before '22 February'
* remove "which [...] modes" in the first line. massive confusing run-on and the mode is explained in its own article anyway